opds.py

```diff
@@ -192,14 +169,14 @@ def categories(cls, work):
         ):
             if value:
                 appeal = dict(term=schema_url + name, label=name)
-                weight_field = "{%s}ratingValue" % schema_ns
+                weight_field = "{%s}ratingValue" % AtomFeed.SCHEMA_NS
```

Comment:
Might also be nice to shortcut this sort of thing as `AtomFeed.schema_("ratingValue")`